### PR TITLE
JakartaEE modules

### PR DESCRIPTION
**Pull request by lukaszlenart**
Moves JakartaEE related modules into a dedicated subdirectory

Depends on #809

### CODE REVIEW

**Comment by jdyer1** [comment]
I agree this is the right thing to do.  We shouldn't break dev experience with their IDEs!

**Review by kusalk — APPROVED** [review]

File: jakarta/velocity-tools-jsp-jakarta/pom.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                                <version>${velocity-tools.version}</version>
+                            </artifact>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <dependencies>
+        <dependency>
```
[the remark this answers is not in the document]
Comment (lukaszlenart) [reply jakarta/velocity-tools-jsp-jakarta/pom.xml:72]: Looks like a valid dependency
> [ERROR] import org.apache.velocity.tools.ToolContext;

File: pom.xml [hunk 2]
```diff
@@ -305,10 +303,10 @@
                         <useDefaultExcludes>true</useDefaultExcludes>
                         <addDefaultLicenseMatchers>false</addDefaultLicenseMatchers>
                         <licenses>
-                            <licens implementation="org.apache.rat.analysis.license.ApacheSoftwareLicense20" />
+                            <licens implementation="org.apache.rat.analysis.license.ApacheSoftwareLicense20"/>
```
[the remark this answers is not in the document]
Comment (lukaszlenart) [reply pom.xml:306]: Right, it's a typo but it doesn't matter as this an element of an array. Yet I fixed it :)

File: jakarta/velocity-tools-jsp-jakarta/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kusalk) [reply jakarta/velocity-tools-jsp-jakarta/pom.xml:72]: Oh I see, it was previously included transitively by `velocity-tools-view`, makes sense

**Review by kusalk — APPROVED** [review]

**Pull request merged**